introspection-engine/connectors/sql-introspection-connector/src/introspection.rs

```diff
@@ -99,14 +99,33 @@ pub fn introspect(
         if let (Some(f), Some(s)) = (table.foreign_keys.get(0), table.foreign_keys.get(1)) {
             let is_self_relation = f.referenced_table == s.referenced_table;
 
-            fields_to_be_added.push((
-                s.referenced_table.clone(),
-                calculate_many_to_many_field(f, table.name[1..].to_string(), is_self_relation),
-            ));
-            fields_to_be_added.push((
-                f.referenced_table.clone(),
-                calculate_many_to_many_field(s, table.name[1..].to_string(), is_self_relation),
-            ));
+            for (fk, opposite_fk) in &[(f, s), (s, f)] {
```

Comment:
this might be readable if extracted to a function (with a long argument list), wdyt?